x-pack/plugins/apm/server/lib/search_strategies/correlations/query_histogram_rangesteps.ts

```diff
@@ -41,25 +49,26 @@ export const fetchTransactionDurationHistogramRangesteps = async (
   esClient: ElasticsearchClient,
   params: SearchServiceParams
 ): Promise<number[]> => {
+  const steps = 100;
+
   const resp = await esClient.search(getHistogramIntervalRequest(params));
 
+  if ((resp.body.hits.total as estypes.SearchTotalHits).value === 0) {
+    return getHistogramRangeSteps(0, 1, 100);
+  }
+
   if (resp.body.aggregations === undefined) {
     throw new Error(
-      'fetchTransactionDurationHistogramInterval failed, did not return aggregations.'
+      'fetchTransactionDurationHistogramRangesteps failed, did not return aggregations.'
```

Comment:
Is it `rangeSteps` or `rangesteps`? The file name and function name and error messages don't agree.

Reply:
Good catch! I renamed everything to be consistent `rangeSteps` and `range_steps` in 509a02d.